### PR TITLE
Add notebook platforms to AutoGluon landing page

### PR DESCRIPTION
**Pull request by gidler**
### Description of Changes
This set of changes adds buttons to the AutoGluon landing page that point users toward running AutoGluon tutorials on Jupyter Notebook servers/platforms such as Google Colab and SageMaker Studio Lab.

#### Background
The SageMaker Studio Lab team and AutoGluon have been pursuing a collaboration where AutoGluon informs our users of the ability to run AutoGluon on Studio Lab, and Studio Lab incorporates AutoGluon into their kernels and website messaging. One part of this collaboration was creating [a repo of AutoGluon tutorials](https://github.com/gidler/autogluon-tutorials) that include buttons that launch the tutorial on Colab and Studio Lab. This change adds pointers to those tutorials from the AutoGluon landing page.

Since AutoGluon is an Open Source project and we want to encourage collaboration from other organizations, we intentionally decided to put Google Colab and Studio Lab on equal footing in our messaging.

AutoGluon is planning to do a website revamp in the near future, but based on timeline requests from the Studio Lab team, we decided it was expedient to make this small change to our website now and then also include the necessary changes to the new website when it is released.

#### Summary of Changes
- add new section to bottom of landing page `docs/index.rst` that has buttons/links to running AutoGluon tutorials on different notebook servers/platforms (the links take you [here](https://github.com/gidler/autogluon-tutorials/blob/main/welcome-notebook.ipynb), [here](https://colab.research.google.com/github/gidler/autogluon-tutorials/blob/main/welcome-notebook.ipynb) or [here](https://studiolab.sagemaker.aws/import/github/gidler/autogluon-tutorials/blob/main/welcome-notebook.ipynb))
- create new file `docs/static/platforms.html` that lays out the buttons. I resorted to implementing this in html because I could not get a good looking result using plain rst with the version of d2l-book AutoGluon uses. I found the image and card constructs in rst to be too restrictive. It appears the D2L book took [this same approach](https://github.com/d2l-ai/d2l-en/blob/master/static/frontpage/frontpage.html#L352-L388).
- copy and check-in png files (from [D2L website](https://github.com/d2l-ai/d2l-en/tree/master/static/frontpage/_images)) for button icons for running Jupyter on a laptop, SageMaker Studio Lab and Google Colab.
- update `build_doc.sh` and github workflow_scripts to include copying images as static resources when website is built
 

#### Screenshots
Website as it is rendered on my laptop browser:
https://user-images.githubusercontent.com/9584334/180495985-fd71d1d0-a16c-4ffb-9af4-4df21025e79e.jpg

Website as it would be seen on a small screen device like a smartphone:
https://user-images.githubusercontent.com/9584334/180495965-9bf2faf6-ddec-402e-94cc-90df982a604b.jpg

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/static/platforms.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+.running-item p {
+    padding: 20px;
+}
+
+</style>
+</head>
+<div class="content">
+	<div class = "mdl-grid running">
+		<div class="running-item">
+			<a href="https://github.com/gidler/autogluon-tutorials/blob/main/welcome-notebook.ipynb">
```
Comment (gradientsky) [inline docs/static/platforms.html:28]: Do we expect `ipynb` files to live in the other repo? What's the long-term strategy for these?

File: docs/static/platforms.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+				<img src="./_static/laptop_jupyter.png"/>
+				<p>Run<br>locally</p>
+		</div>
+		<div class="running-item">
+			<a href="https://colab.research.google.com/github/gidler/autogluon-tutorials/blob/main/welcome-notebook.ipynb">
```
Comment (gradientsky) [inline docs/static/platforms.html:33]: ditto

File: docs/static/platforms.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+				<img src="./_static/colab.png" />
+				<p>Google<br>Colab</p>
+			</a>
+		</div>
+		<div class="running-item">
+			<a href="https://studiolab.sagemaker.aws/import/github/gidler/autogluon-tutorials/blob/main/welcome-notebook.ipynb">
```
Comment (gradientsky) [inline docs/static/platforms.html:39]: ditto

File: docs/static/platforms.html (hunk 1 above)
Comment (goldmermaid) [reply docs/static/platforms.html:28]: For sagemaker studio lab, the notebooks/ipynb are more static. (e.g. in D2L, we have a [separate git repo](https://github.com/d2l-ai/d2l-pytorch-sagemaker-studio-lab) for hosting studio lab notebooks.)

Comment (gidler) [reply docs/static/platforms.html:28]: In a previous conversation Nick suggested I make this new repo to host the notebooks. In the future we may (should) find a better place for them to live long term, but my understanding is creating a new awslab project is prohibitively difficult. Happy to discuss more

File: docs/static/platforms.html (hunk 2 above)
Comment (gidler) [reply docs/static/platforms.html:33]: see answer above

File: docs/static/platforms.html (hunk 3 above)
Comment (gidler) [reply docs/static/platforms.html:39]: see answer above

**Review by gradientsky — APPROVED** [review]

**Review by Innixma — APPROVED** [review]
This is great as an initial PR adding the buttons! We will want to think about how we ideally do this in future PRs, likely as part of the website refactor to pure Sphinx. Probably a combination of making the buttons more visible (including present in each tutorial), telling the user when a tutorial requires GPU, and refining the ipynb welcome page to be easier to navigate to other tutorials. Still, this is an awesome start!

**Pull request merged**